tests/integration/ambient/baseline_test.go

```diff
@@ -2422,7 +2422,7 @@ func buildQuery(src, dst echo.Instance) prometheus.Query {
 	destns := dst.NamespaceName()
 
 	labels := map[string]string{
-		"reporter":                       "destination",
+		"reporter":                       "waypoint",
```

Comment:
we fixed a bunch of labels, seems like we should have more changes here @keithmattix

Reply:
Maybe need to add some more(new) labels

Reply:
Yeah there should be more things, but I don't see unknown checks below. Is the test ignoring them?